frontend/src/App.vue

```diff
@@ -8,5 +8,10 @@
 </template>
 
 <script setup lang="ts">
+import axios from 'axios'
 import FooterComponent from './components/FooterComponent.vue'
+import router from './router'
+axios.get(`${import.meta.env.VITE_API_URL}/projects`).then(() => {
```

Comment:
- you should setup a constants object as we'll need to define some stuff, including the backend base URI
- we want to be able to change most of those at runtime (so after the build process) using environment variables. Check and replicate how zimfarm does this (it creates a JSON file in entrypoint)
- This check and jump should not be in `App`. it's a feature of the Home view